src/components/category-selector/CategorySelector.scss

```
@@ -0,0 +1,65 @@
+@import '../../styles/variables';
+
+/**************************************
+ * Category Selector
+ **************************************/
+
+.bdl-CategorySelector {
+    display: flex;
+    flex-grow: 1;
+    justify-content: center;
+    height: 8 * $bdl-grid-unit;
+
+    .bdl-CategorySelector-links {
+        height: 8 * $bdl-grid-unit;
+        overflow: hidden;
+    }
+
+    .bdl-CategorySelector-pill {
+        margin-right: 2 * $bdl-grid-unit;
+    }
+
+    .bdl-CategorySelector-pill,
+    .bdl-CategorySelector-more-label {
+        display: inline-block;
+        height: 8 * $bdl-grid-unit;
+        padding: 2 * $bdl-grid-unit 4 * $bdl-grid-unit;
+        font-weight: bold;
+        font-size: $bdl-fontSize;
+        line-height: 4 * $bdl-grid-unit;
+        white-space: nowrap;
+        text-align: center;
+        border: 1px solid transparent;
+        border-radius: 4 * $bdl-grid-unit;
+        outline: none;
+        cursor: pointer;
+
+        &:hover,
+        &:focus {
+            padding: 2 * $bdl-grid-unit 4 * $bdl-grid-unit;
+            font-weight: bold;
+            background-color: $bdl-gray-05;
+            border: 1px solid transparent;
```

Can you remove the values here that are the same as the base? If someone needs to override those values, they would need to add the :focus and :hover states, too.

```suggestion
            background-color: $bdl-gray-05;
```

---

Nevermind, this is _because_ of the button component doing that in the first place